qa/suites/fs/workload/tasks/5-workunit/kernel_untar_build.yaml

```diff
@@ -4,6 +4,15 @@ overrides:
       mds:
         - "mds.dir_split"
 tasks:
+- install:
```

Comment:
@dparmar18 Is this change related?

Reply:
yes these packages are very important, else the kernel untar tests would fail

Reply:
Hmmm.. why aren't those failing now. Needs explanation in commit.

Reply:
because the suite that has untar tests also install these packages e.g. `qa/suites/fs/mixed-clients`, symlink to https://github.com/ceph/ceph/blob/main/qa/cephfs/begin/0-install.yaml can be found in it

Reply:
ok ill update it

Reply:
wait, instead of making change to a generic yaml file, i can add these packages to `0-nautilus.yaml` and `0-pacific.yaml`